src/CmsAdmin/CategoryController.php

```diff
@@ -361,17 +361,18 @@ public function sortAction(Request $request)
         if (!is_array($order) || empty($order)) {
             return 'Sortowanie nie powiodło się';
         }
-        //sortowanie
+        //sorting
         foreach ($order as $order => $id) {
```

Comment:
Przydałby się np. phpdoc z "array shape". Dobrze rozumiem że to jest `[ $itemOrder => $itemId, ...]`?
Może nazwa bardziej `$orderedIds`?

Reply:
i to nadpisanie zmiennej w foreachu jest deko mylące, tu $order i tam $order.